Automattic/vip-go-mu-plugins
# Search: Don't try to validate index settings for index versions that don't exist

## Changelog Description


### Plugin Updated: Search

Update to search settings validation job to skip index versions not initialized in ElasticSearch.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

First you need to create an index that only lives in DB not in ES:
```
wp shell
$indexable = \ElasticPress\Indexables::factory()->get('post');
$versioning = new \Automattic\VIP\Search\Versioning();

$versions = $versioning->get_versions( $indexable );
$versions[99] = [ 'number' => 99, 'active' => false];

$versioning->update_versions($indexable, $versions);
```

Add debug line: - search/includes/classes/class-health.php:740
`var_dump('unhealthy', $unhealthy);`

Then running the check:
wp  eval "do_action('vip_search_settings_health');"

Before change it would find an issue after the change all should be good.
